pkg/api/v1/types.go

```diff
@@ -1960,14 +1960,30 @@ type PodLogOptions struct {
 
 	// The container for which to stream logs. Defaults to only container if there is one container in the pod.
 	Container string `json:"container,omitempty"`
-
-	// Follow the log stream of the pod.
-	// Defaults to false.
+	// Follow the log stream of the pod. Defaults to false.
 	Follow bool `json:"follow,omitempty"`
-
-	// Return previous terminated container logs.
-	// Defaults to false.
+	// Return previous terminated container logs. Defaults to false.
 	Previous bool `json:"previous,omitempty"`
+	// A relative time duration to the current time from which to show logs. If this value
+	// preceeds the time a pod was started, only logs since the pod start will be returned.
+	// If this value is in the future, no logs will be returned.
+	// Only one of sinceDuration or sinceTimestamp may be specified.
+	SinceDuration string `json:"sinceDuration,omitempty"`
```

Comment:
Duration in what units? time.Duration? Everywhere else in the API we currently use seconds.
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#naming-conventions